tests/framework/ReleaseControllerTest.php

```
@@ -54,4 +54,60 @@ public function testResortChangelogConvertsFixEntriesToBugsAndSortsThem(): void
+        file_put_contents(
+            $basePath . '/extensions/apidoc/CHANGELOG.md',
+            "Yii Framework 2 apidoc extension Change Log\n"
+            . "===========================================\n\n"
+            . "4.0.0 under development\n"
+            . "-----------------------\n\n"
+            . "- Bug #1: Test change (samdark)\n"
+        );
+
+        $controller = $this->createController();
+        $controller->basePath = $basePath;
+        $method = new \ReflectionMethod($controller, 'getReleaseVersions');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '73', column '23'). (undefined)

(MissingImport)